cultivation/scripts/flashcore/cli/main.py

```
@@ -0,0 +1,221 @@
+        resolve_path=True,
+    ),
+):
+    """Export flashcards to Markdown files, one file per deck."""
+    final_output_dir = output_dir or settings.export_dir
+    console.print(f"Exporting flashcards to [cyan]{final_output_dir}[/cyan]...")
+    try:
+        with FlashcardDatabase() as db:
+            export_to_markdown(db=db, output_dir=final_output_dir)
+    except (FlashcardDatabaseError, IOError) as e:
+        console.print(f"[bold]An error occurred during export: {e}[/bold]")
+        raise typer.Exit(code=1)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)